graph/ops_convolve_test.go

```diff
@@ -240,3 +240,29 @@ func TestGradientConvolve(t *testing.T) {
 			tensors.FromScalarAndDimensions(0.0, 2, 2, 3, 6).Value(),
 		})
 }
+
+func TestConvolveForwardPass(t *testing.T) {
+	testFuncOneInput(t, "Convolve identity kernel with FeatureGroupCount and BatchGroupCount",
+		func(g *Graph) (input, output *Node) {
+			// Create 3x3 input with channels first (batch=1, channels=1)
+			input = Ones(g, MakeShape(dtypes.Float32, 1, 1, 3, 3))
+
+			// Create identity-like 3x3 kernel with proper reshaping
+			kernel := Const(g, [][][][]float32{
+				{{
+					{1.0, 0.0, 0.0},
+					{0.0, 1.0, 0.0},
+					{0.0, 0.0, 1.0},
+				}},
+			})
+
+			// Build convolution with valid padding, feature group count, and batch group count
+			output = Convolve(input, kernel).
+				ChannelsAxis(images.ChannelsFirst).
+				NoPadding().
+				FeatureGroupCount(1).
```

Comment:
It would be nice an example where `FeatureGroupCount != 1` and (maybe another?) example where `BatchGroupCount != 1`.